serverAkkaHttp/src/main/scala/RouterDerivation.scala

```diff
@@ -14,12 +15,22 @@ object RouterDerivationMacro extends RouterDerivationMacro {
     import c.universe._
     val tpe = weakTypeOf[A]
 
+    //check only annotations of path type
```

Comment:
This kind of comment calls out for "introduce method" to improve readability. ;-)